source/install.rst

```diff
@@ -1,599 +0,0 @@
-    $ git clone https://github.com/pyscf/dftd3 /home/abc/dftd3
-    $ echo /home/abc/doci > /home/abc/.pyscf_ext_modules
-    $ echo /home/abc/dftd3 >> /home/abc/.pyscf_ext_modules
-    $ export PYSCF_EXT_PATH=/home/abc/semiempirical:/home/abc/.pyscf_ext_modules
-
-  Using this definition of `PYSCF_EXT_PATH`, the three extension
-  submodules (semiempirical, doci, dftd3) are loaded when PySCF is
-  imported, and you don't have to use a Python virtual environment.
-
-Once the extension modules have been correctly installed (with any of
-the methods shown above), you can use them as regular submodules
-developed inside the pyscf main project::
-
-    >>> import pyscf
-    >>> from pyscf.semiempirical import MINDO3
-    >>> mol = pyscf.M(atom='N 0 0 0; N 0 0 1')
-    >>> MINDO(mol).run()
-
-Common examples
-===============
-... NAO
-... ---
-... The :mod:`nao` module includes basic functions for numerical atomic
-orbitals (NAO) and NAO-based TDDFT methods.  This module was
-contributed by Marc Barbry and Peter Koval. More details of :mod:`nao`
-can be found in
-https://github.com/pyscf/nao/blob/master/README.md. This module can be
-installed with::
-...    $ pip install https://github.com/pyscf/nao
-
-
-DMRG solvers
-------------
-
-Density matrix renormalization group (DMRG) theory is a powerful
-method for solving ab initio quantum chemistry problems. PySCF can be
-used with three implementations of DMRG: Block
-(https://sanshar.github.io/Block), block2
-(https://block2.readthedocs.io/en/latest), and CheMPS2
-(http://sebwouters.github.io/CheMPS2/index.html).
-
-`Installing Block <https://sanshar.github.io/Block/build.html>`_ requires a C++11
-compiler.  If C++11 is not supported by your compiler, you can
-download the precompiled Block binary from https://sanshar.github.io/Block/build.html.
-
-``block2`` can be easily installed via ``pip install block2`` or ``pip install block2-mpi``,
-or `building from source <https://block2.readthedocs.io/en/latest/user/installation.html>`_.
-
-Before using Block or CheMPS2, you need create a configuration file
-``pyscf/dmrgscf/settings.py`` (as shown by settings.py.example) to
-store the path where the DMRG solver was installed.
-
-
-TBLIS
------
-
-`TBLIS <https://github.com/devinamatthews/tblis>`_ provides a native
-algorithm for performing tensor contraction for arbitrarily
-high-dimensional tensors. The native algorithm in TBLIS does not need
-to transform tensors into matrices by permutations, then call BLAS for
-the the matrix contraction, and back-permute the results. This means
-that tensor transposes and data moves are largely avoided by TBLIS. This
-leads to speedups in many correlated quantum chemistry methods in PySCF, such as
-the coupled cluster methods.
-The interface to TBLIS offers an efficient implementation for
-:func:`numpy.einsum` style tensor contraction.  The tblis-einsum
-plugin can be enabled with::
-
-  $ pip install pyscf-tblis
-
-Troubleshooting
-***************
```

Comment:
It seems the troubleshooting section is not covered by new install.md. Should it be retained?